README.md

```diff
@@ -139,3 +188,9 @@ data to be handled efficiently.
  * Needs plenty of robust testing: huge datasets, etc...
  * More thorough benchmarking
  * Your suggestions for use-cases I haven't thought of
+
+
+# Credits and contributions
+
+Original idea, design and implementation: [Peter Bourgon][https://github.com/peterbourgon]
+Other collaborations: [Javier Peletier][https://github.com/jpeletier] ([Epic Labs][https://www.epiclabs.io])
```

Comment:
Pretty sure these Markdown links should be `[Name](url)`? And please add a period at the end of each line.